test/unit/mongo_logger.test.ts

```diff
@@ -486,7 +493,11 @@ describe('class MongoLogger', function () {
         context('when mongodbLogPath is set to valid client option', function () {
           for (const validEnvironmentOption of validEnvironmentOptions) {
             for (const validValue of validClientOptions) {
-              it(`{environment: "${validEnvironmentOption}", client: "${validValue}"} uses the value from the client options`, function () {
+              it(`{environment: "${validEnvironmentOption}", client: ${
+                typeof validValue === 'object'
+                  ? 'new ' + validValue.constructor.name + '(...)'
+                  : '"' + validValue.toString() + '"'
+              }} uses the value from the client options`, function () {
```

Comment:
Emits a more useful test output message. Would emit `[object Object]` when passed in a writable. now emits `new Writable(...)`